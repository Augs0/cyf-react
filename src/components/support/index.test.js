import React from 'react';
import { shallow } from 'enzyme';

import Support from './';

describe('Support Component', () => {
  it('renders', () => {
    expect(shallow(<Support />)).toMatchSnapshot();
  });
});
