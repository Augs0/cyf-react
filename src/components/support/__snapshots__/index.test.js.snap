// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Support Component renders 1`] = `
<div
  className="section-container"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-6 col-md-6"
      >
        <h2>
          Want to Contribute?
        </h2>
      </div>
      <div
        className="col-sm-6 col-md-6 block-box help-section"
      >
        <h3
          className="text-left"
        >
          We are looking for
        </h3>
        <br />
        <ul
          className="text-left"
        >
          <li>
            <span
              className="list-icon"
            >
              <img
                alt="Space"
                src="conference-hall.svg"
              />
            </span>
            Space
          </li>
          <li>
            <span
              className="list-icon"
            >
              <img
                alt="Mentors"
                src="code.svg"
              />
            </span>
            Mentors (Javascript)
          </li>
          <li>
            <span
              className="list-icon"
            >
              <img
                alt="Used Laptops"
                src="laptop.svg"
              />
            </span>
            Used Laptops
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
